app/controllers/api/v1/brands_controller.rb

```diff
@@ -7,4 +7,7 @@ def show
     select_brand = Brand.find(params[:id])
     render json: {brand: select_brand, pizzas: select_brand.pizzas}
   end
+
+  def new 
+  end
```

Comment:
this is not needed. please remove! there is no point in empty routes for api controllers